core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -880,20 +846,6 @@
           + " general.vfs.context.classpath.<name>.delegation=post, where <name> is"
           + " your context name. If delegation is not specified, it defaults to loading"
           + " from parent classloader first."),
-  @Interpolated
-  VFS_CLASSLOADER_CACHE_DIR(AccumuloVFSClassLoader.VFS_CACHE_DIR,
-      "${java.io.tmpdir}" + File.separator + "accumulo-vfs-cache-${user.name}",
-      PropertyType.ABSOLUTEPATH,
-      "Directory to use for the vfs cache. The cache will keep a soft reference"
-          + " to all of the classes loaded in the VM. This should be on local disk on"
-          + " each node with sufficient space. It defaults to"
-          + " ${java.io.tmpdir}/accumulo-vfs-cache-${user.name}"),
-
-  @Interpolated
-  @Experimental
-  GENERAL_MAVEN_PROJECT_BASEDIR(AccumuloClassLoader.MAVEN_PROJECT_BASEDIR_PROPERTY_NAME,
-      AccumuloClassLoader.DEFAULT_MAVEN_PROJECT_BASEDIR_VALUE, PropertyType.ABSOLUTEPATH,
```

Comment:
Like the VFS property, this one might still be in use via the referenced `AccumuloClassLoader.MAVEN_PROJECT_BASEDIR_PROPERTY_NAME` property, but here for documentation. That said, we could probably remove it... I don't know any developers using it, and it's only here for developers.

Reply:
Ok, I will leave this as is then